packages/bp/src/core/config/botpress.config.ts

```diff
@@ -7,12 +7,14 @@ import { IncidentRule } from '../health/alerting-service'
 
 export type BotpressCondition = '$isProduction' | '$isDevelopment'
 
-export interface ModuleConfigEntry {
+export interface ModuleConfigEntry
+{
```

Comment:
Please fix eslint / prettier, the bracket should be on the same line

Reply:
Yes It's fixed, config problem :)